packages/hoppscotch-data/src/rest/v/11.ts

```diff
@@ -0,0 +1,145 @@
+import {
+  HoppRESTAuthBasic,
+  HoppRESTAuthBearer,
+  HoppRESTAuthInherit,
+  HoppRESTAuthNone,
+} from "./1"
+import { HoppRESTAuthAPIKey } from "./4"
+import { AuthCodeGrantTypeParams, HoppRESTAuthAWSSignature } from "./7"
+import {
+  ClientCredentialsGrantTypeParams as ClientCredentialsGrantTypeParamsOld,
+  HoppRESTAuthDigest,
+  PasswordGrantTypeParams,
+} from "./8"
+import { ImplicitOauthFlowParams } from "./3"
+import { z } from "zod"
+
+import { V10_SCHEMA } from "./10"
+import { defineVersion } from "verzod"
+import {
+  HoppRESTResponseOriginalRequest as HoppRESTResponseOriginalRequestOld,
+  HoppRESTRequestResponse as HoppRESTRequestResponseOld,
+} from "./9"
+
+export const ClientCredentialsGrantTypeParams =
+  ClientCredentialsGrantTypeParamsOld.extend({
+    clientAuthentication: z.enum(["AS_BASIC_AUTH_HEADERS", "IN_BODY"]),
+  })
+
+export const HoppRESTAuthOAuth2 = z.object({
+  authType: z.literal("oauth-2"),
+  grantTypeInfo: z.discriminatedUnion("grantType", [
+    AuthCodeGrantTypeParams,
+    ClientCredentialsGrantTypeParams,
+    PasswordGrantTypeParams,
+    ImplicitOauthFlowParams,
+  ]),
+  addTo: z.enum(["HEADERS", "QUERY_PARAMS"]).catch("HEADERS"),
+})
+
+export type HoppRESTAuthOAuth2 = z.infer<typeof HoppRESTAuthOAuth2>
+
+export const HoppRESTAuth = z
+  .discriminatedUnion("authType", [
+    HoppRESTAuthNone,
+    HoppRESTAuthInherit,
+    HoppRESTAuthBasic,
+    HoppRESTAuthBearer,
+    HoppRESTAuthOAuth2,
+    HoppRESTAuthAPIKey,
+    HoppRESTAuthAWSSignature,
+    HoppRESTAuthDigest,
+  ])
+  .and(
+    z.object({
+      authActive: z.boolean(),
+    })
+  )
+
+export type HoppRESTAuth = z.infer<typeof HoppRESTAuth>
+
+export const HoppRESTResponseOriginalRequest =
+  HoppRESTResponseOriginalRequestOld.extend({
+    v: z.literal("3"),
+    auth: HoppRESTAuth,
+  })
```

Comment:
The `HoppRESTResponseOriginalRequest` can include a `body` field from `v10`, accommodating the bulk mode preference. This accounts for the issue whereby a request with a bulk mode preference for the body defaults to the standard view when saving a response example, instead of adhering to the original request's bulk edit mode preference.